components/bsx/Gallery/Item/AvailableActions.vue

```diff
@@ -183,6 +183,12 @@ export default class AvailableActions extends mixins(
     }
     this.$consola.log(typeof value, value)
     this.meta = value
+    // ad-hoc fix for empty input value
+    if (this.meta === '0') {
+      this.isBalanceInputValid = false
+    } else {
+      this.isBalanceInputValid = balanceInputComponent.checkValidity()
+    }
```

Comment:
Before calling `checkValidity`, we should make sure `balanceInputComponent instanceof BalanceInput`.

Or there will be some error.

Reply:
made a duplicate check, and removed an unnecessary one.
Also, add a guard for BlanceInput value check.

btw, that's why viki mentioned we should avoid `ref`...